src/components/markdown-renderer/replace-components/image/frame.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { Fragment, useState } from 'react'
+import { Modal } from 'react-bootstrap'
+import { ForkAwesomeIcon } from '../../../common/fork-awesome/fork-awesome-icon'
+
+export const Frame: React.FC<React.ImgHTMLAttributes<HTMLImageElement>> = ({ alt, ...props }) => {
+  const [showFullscreenImage, setShowFullscreenImage] = useState(false)
+
+  return (
+    <Fragment>
+      <img alt={alt} {...props} onClick={() => setShowFullscreenImage(true)}/>
```

Comment:
There is no indicator for the user that says "click me to open a modal". It would be enough, if the image gets a new class, that sets the cursor to pointer.

Reply:
done